partiql-plan/src/main/kotlin/org/partiql/plan/v1/PartiQLPlan.kt

```diff
@@ -0,0 +1,16 @@
+package org.partiql.plan.v1
+
+public interface PartiQLPlan {
+
+    public fun getStatement(): Statement
+}
+
+/**
+ * TODO INTERNALIZE ME
+ * @property statement
+ */
+public class PartiQLPlanImpl(
```

Comment:
Why does this need to be public in this PR?

Reply:
It technically doesn't need to be in _this_ PR